client/src/app/primitives/Button.less

```diff
@@ -1,54 +1,21 @@
 :local(.Button) {
+  box-sizing: border-box;
+  border-radius: 3px;
   padding: 5px;
-  background: white;
-  border: solid 1px #ddd;
-  border-radius: 2px;
+  border: solid 1px transparent;
+  background: transparent;
+  vertical-align: middle;
+  font-size: 14px;
   outline: none;
-
-  &:hover {
-    background: #eee;
-  }
-
-  &.primary {
-    background: #489d12;
-    color: white;
-    border-color: #489d12;
-
-    &:hover {
-      background: #2F8400;
-      border-color: #2F8400;
-    }
-  }
-
-  &.disabled {
```

Comment:
Is there a reason you replaced the locally scoped style with a style global `.disabled` declaration?

For the sake of understandability I'd keep the local style override pattern:

```less
:local(.someCls) {
  &.someModifierCls { ... }
}
```

Referring to it in code:

```
return <div className={ classNames(style.someCls, { someModifierCls } }>;
```